java/src/test/java/com/mlt/decoder/MltDecoderTest.java

```diff
@@ -1,130 +1,136 @@
 package com.mlt.decoder;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
 import com.mlt.TestSettings;
 import com.mlt.TestUtils;
 import com.mlt.converter.ConversionConfig;
 import com.mlt.converter.FeatureTableOptimizations;
 import com.mlt.converter.MltConverter;
 import com.mlt.converter.mvt.ColumnMapping;
-import com.mlt.converter.mvt.MapboxVectorTile;
 import com.mlt.converter.mvt.MvtUtils;
-import com.mlt.metadata.tileset.MltTilesetMetadata;
 import java.io.IOException;
 import java.nio.file.Paths;
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
-import org.apache.commons.lang3.tuple.Triple;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 
-@FunctionalInterface
-interface TriConsumer<A, B, C> {
-  void apply(A a, B b, C c) throws IOException;
+enum DecoderType {
+  BOTH,
+  SEQUENTIAL,
+  VECTORIZED
 }
 
 public class MltDecoderTest {
 
-  private static Stream<Triple<Integer, Integer, Integer>> bingMapsTileIdProvider() {
+  /* Bing Maps tests --------------------------------------------------------- */
+
+  private static Stream<String> bingProvider() {
     return Stream.of(
-        Triple.of(4, 8, 5), Triple.of(5, 16, 11), Triple.of(6, 32, 22), Triple.of(7, 65, 42));
+        "4-8-5", "4-9-5", "4-12-6", "4-13-6", "5-16-11", "5-17-11", "5-17-10", "6-32-22", "6-33-22",
+        "6-32-23", "6-32-21", "7-65-42", "7-66-42", "7-66-43", "7-66-44");
   }
 
-  @DisplayName("Decode unsorted Bing Maps based vector tiles")
+  @DisplayName("Decode Bing Tiles (Vectorized)")
   @ParameterizedTest
-  @MethodSource("bingMapsTileIdProvider")
-  public void decodeMlTileVectorized_UnsortedBingMaps(Triple<Integer, Integer, Integer> tileId)
-      throws IOException {
-    // TODO: fix -> 5-15-10, 5-17-10, 5-17-11, 7-65-42, 9-259-176
-    var id = String.format("%s-%s-%s", tileId.getLeft(), tileId.getMiddle(), tileId.getRight());
-    testTileVectorized(id, TestSettings.BING_MVT_PATH, false);
+  @MethodSource("bingProvider")
+  public void decodeBingTilesVectorized(String tileId) throws IOException {
+    int numErrors = testTile(tileId, TestSettings.BING_MVT_PATH, DecoderType.VECTORIZED, false);
+    assertEquals(
+        0,
+        numErrors,
+        "There should be no errors in the Bing tiles for "
+            + tileId
+            + " but there were "
+            + numErrors
+            + " errors");
   }
 
-  private static Stream<Triple<Integer, Integer, Integer>> omtTileIdProvider() {
-    return Stream.of(
-        Triple.of(2, 2, 2),
-        Triple.of(3, 4, 5),
-        Triple.of(4, 8, 10),
```

Comment:
My thoughts on this were -> by using int values ​​we can dynamically change the file name, e.g. change the separator of xyz in an easy way as i have tiles lying around that have different separators.   
This change is ok for me, but maybe we can agree on a common file naming structure for all tiles so using "_" or "-" for example

Reply:
Ah, okay. Yes, how about we standardize on `-` as the separator? `z-x-y.mvt`? I can make the change in this PR to rename any non-conforming tiles. Does that sound good?

Reply:
> I can make the change in this PR to rename any non-conforming tiles. Does that sound good?

Actually, it's a rather large change to do the renaming. I'd like to do in a followup PR instead of this one as it will require updating JS side as well.

Reply:
ticketed at https://github.com/maplibre/maplibre-tile-spec/issues/179 to ensure we agree and then I can get a PR going after.

Reply:
Sounds good